api_tests/src/shared.ts

```diff
@@ -310,6 +319,34 @@ export async function getPost(
   return api.client.getPost(form);
 }
 
+export async function getPosts(
```

Comment:
This function is unused.

Reply:
I have new pull requests coming that use it, I wanted you to have the chance to get different branches for different intention in changes.